pkg/operator/ceph/config/conditions.go

```diff
@@ -172,21 +175,25 @@ func conditionMapping(conditions []cephv1.Condition) {
 }
 
 // CheckConditionReady checks whether the cluster is Ready and returns the message for the Progressing ConditionType
-func CheckConditionReady(context *clusterd.Context, namespace, name string) string {
-	cluster, err := context.RookClientset.CephV1().CephClusters(namespace).Get(name, metav1.GetOptions{})
-	if err != nil {
-		logger.Errorf("failed to get cluster %v", err)
-	}
-	if cluster.Status.Conditions != nil && len(conditionMap) == 0 {
-		conditionMapping(cluster.Status.Conditions)
+func CheckConditionReady(context *clusterd.Context, namespace, name, service string) string {
+	if name != "" {
+		cluster, err := context.RookClientset.CephV1().CephClusters(namespace).Get(name, metav1.GetOptions{})
+		if err != nil {
+			logger.Errorf("failed to get cluster %v", err)
+		}
+		if cluster.Status.Conditions != nil && len(conditionMap) == 0 {
+			conditionMapping(cluster.Status.Conditions)
+		}
 	}
 	if conditionMap[cephv1.ConditionReady] == v1.ConditionTrue {
 		return "Cluster is checking if updates are needed"
+	} else if service == "mgr" {
```

Comment:
Why does mgr get a special entry?

Reply:
I was using a different message for mgr.